### ISSUE TITLE
Improve the onboarding experience via matrix.org

### ISSUE THREAD

**Issue by thibaultamartin** [matrix-org/matrix.org#2178]
https://blog.koehntopp.info/2024/02/13/the-matrix-trashfire.html gives the perspective of a person trying Matrix for the first time, and we can do much better!

The main gripes on the matrix.org website are:

- People on macOS miss the install link if they don't open the client details, and can go straight to the app store
- The landing page is too empty and doesn't say much
- The menu goes in responsive mode too early
- The Try Matrix section is too confusing for the general public, and leads to dangerous instances

The rest seems to be related to Element and Element X themselves and is beyond what the matrix.org website can do.

**Comment by MTRNord** [comment]
Additionally, I noticed that currently on the "try matrix now" page the flow is: you get a client and are in the flow, then realise "oh I need a server. What is that?", dive back into the page if you not already lost it and then mess it up only to later realise there was a guide on it.

Instead, it seems to be a better idea to have the download link at the bottom after the explanation. Yes sure it is then further away but it possibly eliminates some confusion in the apps itself.

**Comment by spaetz** [comment]
Given that  MR https://github.com/matrix-org/matrix.org/pull/2179 has already been merged, here my comments that I posted post-merger again:


`**Install Element**`
May I suggest a different wording here? "Install Element" sounds like a high hurdle already and puts me off. And at least for Element-web, that is not needed (even if you decide to install something later). "Use Element" might be a compromise that sounds like the hurdle is much lower (just open it in a webbrowser, hey!).

In the case of mobile, users would likely want to install an app though...

`**Browse clients**`
Only playing a little stupid, but "[Browse clients]" might also be suboptimal. A browse client is e.g. hydrogen or element-web :-). Something like "Discover more clients"?


`**Start chatting**`
More wording nitpicking, sorry. The "Register and start chatting section" has a "Start chatting" button. This should IMHO lead to https://app.element.io/#/register rather than just to some longish manual.
Also the "Read the docs" button uses such a very generic term that I did not know what to expect (the spec?) when clicking it. Perhaps call it "Read client manual" or something?

In the case of mobiles, one could link to the fluffychat web app instead of the element-web register page, which AFAIK has a decent mobile layout? Less hurdles and less intermediary steps and manuals to actually start chatting...

**Comment by BMaxV** [comment]
You're being good sports about this!

Something I would like to add, after trying matrix in a "discord vs. matrix" online debate, is that "rooms" / personal spaces apparently can't be deleted? This is "fine" in the sense that they are effectively inaccessible (and encrypted) and effectively don't exist anymore, but it deserves a mention. Especially for "half admis" / "group leaders" who don't want to set up an entire server, but want a room for their gaming guild / small FOSS project and want to inform themselves and try things.

I don't remember setting up my account, but I managed to do it without been "in the community" much. But i agree with the article that it can be hard?

This path should work but it's a bit inconvenient.

matrix.org -> try matrix (top right corner) -> create account (3rd option) -> https://matrix.org/docs/chat_basics/matrix-for-im/#creating-a-matrix-account probably works? At least the screen shot seems correct. I can't verify because it automatically redirects me to the logged in session.

This is not exactly a matrix issue, but the https://element.io/ homepage "get started" leads to an organization signup, not for individual users.

**Comment by thibaultamartin** [comment]
Interesting insights, since Element is recommended by the Foundation as a safe default, it makes sense to ask Element to make more room for the individuals use case indeed.

We need to sit down with Element marketing and spend some time doing something similar to https://github.com/matrix-org/matrix.org/issues/1502 and https://github.com/matrix-org/matrix.org/issues/1543

**Comment by isotopp** [comment]
I never got that far.

I started with downloading the client. I chose the client that was supported on MacOS, since I hate typing with an on-screen keyboard. That was Element X, in contradiction to what the site says (see screenshot): Element is not available in the App Store, only Element X is.

![Screenshot 2024-02-14 at 18 25 43](https://github.com/matrix-org/matrix.org/assets/182951/5e9c8929-f21d-45a4-996a-1d1ada34a287)

I was unable to create an account in that application. The application said something about Matrix.org, so I went to the Site.

The says nothing on the start page, and also nothing when you scroll down. Scroll up again, find "Try Matrix". Click on this, you are "in a loop", it says "Install Element" (I just did that, downloading Element X from the App Store). Ignore, scroll down more, find something Account related. That's what I want, I click it, and end up on joinmatrix as documented.

I see this has been changed now. That is good.

joinmatrix had no matrix.org in the list of available servers, so I ended up on tchncs.de, which I know provides a good Mastodon experience. In their case, the web frontend ("chat.tchncs.de") and the thing that you need to tell Element X when it speaks about Identity Server ("tchncs.de") differ. Trying chat.tchncs.de or trying the URL from the tchncs.de web interface and pasting that do not work and produce unhelpful errors in Element X. The latter ends up with a message that asks me, an end user, to fix the server config, presented in a Markdown File in Github – WTF.

**Comment by MTRNord** [comment]
> in contradiction to what the site says (see screenshot): Element is not available in the App Store, only Element X is.

Just to clairfy (and your Blog Post also shows that) the website never claimed it is on the app store but rather that macos is supported. Which unfortunately (decided by element) is distributed via a dmg file on their Website instead of the Apple store :/

I do agree however that it would need to be clearer that the client cards are clickable for more information. :/

**Comment by MTRNord** [comment]
yep the ecosystem pages need some unification/overhaul. They have some more issues like titles not fitting or breaking properly and similar.
